### ISSUE TITLE
was the Receiver trained for 2 epochs also?

### ISSUE THREAD

**Issue by Jeiyoon** [SivilTaram/Persona-Dialogue-Generation#22]
Hi thank you for your great works!

i have three questions

1)
I checked the number of epochs for training the Transitter in the paper: two

But I'd like to ask to the number of epochs for training the Receiver

it's because the bleu score during the test (eval_f1.py) is very low (e.g., 0.01)

2)
And could you let me know more details about training PSqure bot (e.g., training epochs)

3) 
Finally, what's the difference of the tasks between the "orignial" and "revised"

not literal meaning, but training  

These things are not described in the paper so I failed to reimplement the test results.

I really appreciate if you let me know more details for getting test results as the paper.

thank you

**Comment by SivilTaram** [comment]
@Jeiyoon Hi, many thanks for your interest on our work!
1. The low BLEU score is normal (since there is only one reference sentence to evaluate), which is also consistent with prior works (Li et al. 2016). Sorry for forgetting the detailed training epochs on Receiver, but it should arrive the Receiver performance reported in our paper (e.g. In original setting, it should be similar to 93.8). You could refer Sec. 5.4 for more details.
2. The training steps last nearly ~2000 dialogues, as written in our paper (Sec. 5.1).
3. In practice, these two settings are trained under difference data configurations (original or revised). The related description can be seen below:
> To make it more challenging, PERSONA-CHAT also provides revised personas by rephrasing, generalizing or specializing the original ones. For example, “I am overweight.” is revised from “I weight 300 pounds.

**Comment by SivilTaram** [comment]
@Jeiyoon We also provide pretrained model weights, which could be used for reproducing our reported results (please see README.md)

**Comment by Jeiyoon** [comment]
@SivilTaram Thanks so much for letting me know!

I've double-checked experimental setup

I trained transmitter and receiver for 2 epochs (original task, not revised)

And I also trained PSquare for 2000 episodes (original task)

But the bleu score is still very low (e.g., 0.005) 

Did i do something wrong? or could you let me know how can i reimplement to get the bleu score (0.75)?

Every metric is fine but just bleu score

Thank you so much

**Comment by SivilTaram** [comment]
@Jeiyoon There must be misleading. Please note that the reported 0.75 is actually 0.75%, which is less than 0.01. You could check it carefully on Table 3.

**Comment by Jeiyoon** [comment]
oh gosh you are right i misread

everything is fine. it's great and novel work

thank you 

have a nice day!

**Comment by SivilTaram** [comment]
@Jeiyoon ☕ Thanks again for your interest!
